[2022-12-06 13:26:37,055] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_mongo_db_dag.LoadPositionsInMongoDB 2022-12-06T13:25:40.483181+00:00 [queued]>
[2022-12-06 13:26:37,063] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_mongo_db_dag.LoadPositionsInMongoDB 2022-12-06T13:25:40.483181+00:00 [queued]>
[2022-12-06 13:26:37,064] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2022-12-06 13:26:37,064] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2022-12-06 13:26:37,064] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2022-12-06 13:26:37,077] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): LoadPositionsInMongoDB> on 2022-12-06T13:25:40.483181+00:00
[2022-12-06 13:26:37,080] {standard_task_runner.py:52} INFO - Started process 759 to run task
[2022-12-06 13:26:37,084] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_mongo_db_dag', 'LoadPositionsInMongoDB', '2022-12-06T13:25:40.483181+00:00', '--job-id', '42', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_mongo_db_dag.py', '--cfg-path', '/tmp/tmpr__3wmr5', '--error-file', '/tmp/tmpmw0_904e']
[2022-12-06 13:26:37,084] {standard_task_runner.py:77} INFO - Job 42: Subtask LoadPositionsInMongoDB
[2022-12-06 13:26:37,122] {logging_mixin.py:104} INFO - Running <TaskInstance: my_mongo_db_dag.LoadPositionsInMongoDB 2022-12-06T13:25:40.483181+00:00 [running]> on host 2e62a70a5623
[2022-12-06 13:26:37,167] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_mongo_db_dag
AIRFLOW_CTX_TASK_ID=LoadPositionsInMongoDB
AIRFLOW_CTX_EXECUTION_DATE=2022-12-06T13:25:40.483181+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-06T13:25:40.483181+00:00
[2022-12-06 13:26:37,206] {logging_mixin.py:104} INFO - debut de chargement des données dans mongodb
[2022-12-06 13:26:37,484] {logging_mixin.py:104} INFO - fin de chargement des données dans mongodb
[2022-12-06 13:26:37,487] {python.py:151} INFO - Done. Returned value was: None
[2022-12-06 13:26:37,495] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=my_mongo_db_dag, task_id=LoadPositionsInMongoDB, execution_date=20221206T132540, start_date=20221206T132637, end_date=20221206T132637
[2022-12-06 13:26:37,514] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-12-06 13:26:37,536] {local_task_job.py:151} INFO - Task exited with return code 0
